app/views/my_facilities/_facilities_tables_nav.html.erb

```diff
@@ -4,21 +4,17 @@
 
 <div class="d-flex fd-column fd-lg-row justify-lg-between mb-24px d-print-none sub-menu">
   <div class="d-flex order-2 order-lg-1">
-    <p class="mr-12px mr-lg-24px mb-0px pb-8px fs-16px fs-lg-20px fw-bold td-none <%= params[:action] == "bp_controlled" ? "c-red bb-2px bb-red" : "c-grey-dark" %>">
-      <%= link_to(my_facilities_bp_controlled_path(preserve_query_params(request.query_parameters, ["zone", "size", "facility_group"])), class: "#{active_action?("bp_controlled")}") do %>
+    <nav class="nav">
```

Comment:
Since this is the tabbed navigation, let's change this classname to `tabbed-nav`, making it clearly distinct from our main nav.

Reply:
This is the standard name for bootstrap "nav". Can we keep this consistent with bootstrap?